listeners/measured.go

```diff
@@ -37,7 +37,7 @@ func (l *stateAwareMeasuredListener) Accept() (c net.Conn, err error) {
 	if err != nil {
 		return nil, err
 	}
-	fs := make(chan *measured.Stats)
+	fs := make(chan *measured.Stats, 1)
```

Comment:
I'm making this a buffered channel just in case the final stat submission is blocked waiting on the reader from this channel